src/serialization/index.ts

```
@@ -0,0 +1,691 @@
+		}
+		
+		html += '>'
+		
+		if (options.includeTextContent !== false && element.textContent) {
+			html += escapeHTML(element.textContent)
+		}
+		
+		html += `</${element.tagName.toLowerCase()}>`
+		
+		return html
+	})
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)